Tests/AutoPilotTests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AutoPilotTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import XCTest
+@testable import KeyboardSupport
+
+class AutoPilotTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var viewController: AutoNavigatorViewController! = nil
+    private var window: UIWindow!
+    
+    // MARK: - Tests
+    
+    override func setUp() {
+        super.setUp()
+        viewController = UIStoryboard.init(name: "Main", bundle: Bundle(for: AutoPilotTests.self)).instantiateViewController(withIdentifier: "AutoNavigatorViewController") as? AutoNavigatorViewController
+        
+        viewController.loadViewIfNeeded()
+        _ = viewController.view
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        viewController = nil
+    }
+    
+    func test_KeyboardAutoNavigator_AutoPilotDoesNavigateNext() {
+        // Arrange / Act
+        let nextFrom1 = KeyboardAutoNavigator.AutoPilot.nextField(from: viewController.textField1, in: viewController.view)
+        let nextFrom2 = KeyboardAutoNavigator.AutoPilot.nextField(from: viewController.textField2, in: viewController.view)
+        let nextFromView = KeyboardAutoNavigator.AutoPilot.nextField(from: viewController.textView, in: viewController.view)
+        let nextFrom3 = KeyboardAutoNavigator.AutoPilot.nextField(from: viewController.textField3, in: viewController.view)
+        
+        // Assert
+        XCTAssertEqual(nextFrom1!, viewController.textField2)
```

Comment:
To make validation of these a bit easier, we may want to add a quick comment above the `viewController` property in here that describes how that screen is laid out. Something along the lines of:
```swift
    //  [Text Field 1]
    //  [Text Field 2]
    //  [Text View]
    //  [Text Field 3]
```

There may very well be a better way to represent the ordering though.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/d1bf4bf6c0c02a5ea0200b41f829e0c99589c26d